diff --git a/node_modules/react-joyride/dist/index.mjs b/node_modules/react-joyride/dist/index.mjs
index 7a1a011..f2d4e73 100644
--- a/node_modules/react-joyride/dist/index.mjs
+++ b/node_modules/react-joyride/dist/index.mjs
@@ -1,5 +1,6 @@
 import React, { useReducer, useEffect, useMemo, useCallback } from 'react';
-import ReactDOM from 'react-dom';
+import { createPortal } from 'react-dom';
+import { createRoot } from 'react-dom/client';
 import { autoUpdate, useFloating, offset, flip, shift, inline, arrow } from '@floating-ui/react';
 import { isElement, isNumber, isString, get, isObject, isFunction, deepmerge } from 'lodash';
 import createStore from 'zustand';
@@ -1318,17 +1319,17 @@
 
     _this.portal = portal;
 
-    ReactDOM.render(children, portal);
+    createRoot(portal).render(children);
   };
 
   _this.unmount = function () {
     if (!_this.portal) {
       return;
     }
-
-    ReactDOM.unmountComponentAtNode(_this.portal);
+    
     cleanReactTree(_this.portal);
     document.body.removeChild(_this.portal);
+
     _this.portal = null;
   };
 
@@ -1487,7 +1488,7 @@
       }
 
       return /*#__PURE__*/React.createElement(Portal, {
-        children: /*#__PURE__*/ReactDOM.createPortal(content, document.body)
+        children: /*#__PURE__*/createPortal(content, document.body)
       });
     }
 
@@ -1498,7 +1499,7 @@
       }
 
       if (target && canUseDOM) {
-        return /*#__PURE__*/ReactDOM.createPortal(content, getElement(target));
+        return /*#__PURE__*/createPortal(content, getElement(target));
       }
 
       return null;
